content/en/docs/instrumentation/go/sampling.md

```diff
@@ -1,36 +1,43 @@
 ---
 title: Sampling
 weight: 80
-cSpell:ignore: sdktrace
 ---
 
 Sampling is a process that restricts the amount of traces that are generated by
 a system. The exact sampler you should use depends on your specific needs, but
 in general you should make a decision at the start of a trace, and allow the
 sampling decision to propagate to other services.
 
-A sampler needs to be set on the tracer provider when its configured, as
-follows:
+A [`Sampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#Sampler) can
+be set on the tracer provider using the
+[`WithSampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#WithSampler)
+option, as follows:
 
 ```go
-provider := sdktrace.NewTracerProvider(
-    sdktrace.WithSampler(sdktrace.AlwaysSample()),
+provider := trace.NewTracerProvider(
+    trace.WithSampler(trace.AlwaysSample()),
 )
 ```
 
-`AlwaysSample` and `NeverSample` are fairly self-explanatory. Always means that
-every trace will be sampled, the converse holds as true for Never. When you're
-getting started, or in a development environment, you'll almost always want to
-use `AlwaysSample`.
+[`AlwaysSample`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#AlwaysSample)
+and
+[`NeverSample`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#NeverSample)
+are self-explanatory values. `AlwaysSample` means that every trace is sampled,
+while `NeverSample` means that no trace is sampled. When you're getting started,
+or in a development environment, use `AlwaysSample`.
 
 Other samplers include:
 
-- `TraceIDRatioBased`, which will sample a fraction of traces, based on the
-  fraction given to the sampler. Thus, if you set this to .5, half of traces
-  will be sampled.
-- `ParentBased`, which behaves differently based on the incoming sampling
-  decision. In general, this will sample spans that have parents that were
-  sampled, and will not sample spans whose parents were _not_ sampled.
+- [`TraceIDRatioBased`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#TraceIDRatioBased),
+  which samples a fraction of traces, based on the fraction given to the
+  sampler. If you set .5, half of all the traces are sampled.
+- [`ParentBased`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#ParentBased),
+  which behaves differently based on the incoming sampling decision.
+  `ParentBased` samples spans that have parents that were sampled, and doesn't
```

Comment:
nit: can we be consistent about whether the "trace" or the "span" is sampled?  Generally, I would actually prefer "spans" being sampled, but we appear to use "trace is sampled" more often.

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/3318/commits/7d941581f6bbfcb3d8247ffac3c2453f96343a01